src/mono/mono/mini/mini-arm64.c

```diff
@@ -2010,6 +2034,21 @@ mono_arch_set_native_call_context_args (CallContext *ccontext, gpointer frame, M
 		else
 			storage = arg_get_storage (ccontext, ainfo);
 
+#ifdef MONO_ARCH_HAVE_SWIFTCALL
+		if (mono_method_signature_has_ext_callconv (sig, MONO_EXT_CALLCONV_SWIFTCALL)) {
```

Comment:
As in amd64 case, I think all this code can be removed. Here we would need to make `arg_get_storage` correctly return the storage for context regs, since the CallContext doesn't contain all the registers as on amd64.